src/pip/_internal/wheel.py

```diff
@@ -68,15 +68,22 @@ def normpath(src, p):
     return os.path.relpath(src, p).replace(os.path.sep, '/')
 
 
-def rehash(path, blocksize=1 << 20):
-    # type: (str, int) -> Tuple[str, str]
+def hash_file(path, blocksize=1 << 20):
```

Comment:
It would be good to add a simple unit test of this function that you made. It can be added to `tests/unit/test_wheel.py` and can use the `tmpdir` fixture. It doesn't have to be a real wheel file, but maybe a file that you know will loop at least once in the for loop.

And if you've made that test, maybe it would be a simple matter to also add a unit test of the related `rehash()`, which would be a big help. (You could use the same code to make the test file.)

Reply:
Added unit tests with 0fc8dcb